src/main/java/io/pivotal/migration/MigrationClient.java

```diff
@@ -205,7 +207,18 @@ public void createMilestones(List<JiraVersion> versions) {
 			if (version.getReleaseDate() != null) {
 				map.put("due_on", version.getReleaseDate().toString(dateTimeFormatter));
 			}
-			this.getRest().exchange(requestBuilder.body(map), MAP_TYPE);
+			try {
+				getRest().exchange(requestBuilder.body(map), MAP_TYPE);
+			}
+			catch (HttpClientErrorException.UnprocessableEntity ex) {
+				// The milestone already exists on GH, so swallow its error and continue
+				if (ex.getMessage() != null && !ex.getMessage().contains("already_exists")) {
+					throw ex;
+				}
+				else {
+					logger.info("Milestone '{}' exists; ignore", version.getName());
+				}
```

Comment:
This might be more clear, and re-throws if the message is `null`:

```suggestion
			catch (HttpClientErrorException.UnprocessableEntity ex) {
				if (ex.getMessage() != null && ex.getMessage().contains("already_exists")) {
					logger.info("Milestone '{}' exists; ignore", version.getName());
				}
				else {
					throw ex;
				}
```